blog/_posts/2018-09-20-team-barcelonaduo-goodbye.md

```diff
@@ -0,0 +1,108 @@
+---
+title: The end of the beginning
+layout: post
+created_at: Thurs September 20 2018
+permalink: blog/2018-09-20-team-barcelonaduo-goodbye
```

Comment:
Hi @amaliacardenas ! Thanks for the PR. The blog post looks amazing. The only comment I have is all images are missing image-credits and image description. Can you please add them? We'll be ready to publish it after that. :)

Reply:
Hi @nerdyvaishali. I added the missing image credits and image description. Let me know if there is anything else that needs to be added. :)

Reply:
Hi @amaliacardenas ! Your pull request contains 2 blogpost files and bunch of unnecessary images. Can you please remove unnecessary images and file? Thanks